odoo/models.py

```diff
@@ -3134,103 +3163,120 @@ def _fetch_field(self, field):
                 fnames.append(field.name)
         else:
             fnames = [field.name]
-        self._read(fnames)
+        self.fetch(fnames)
 
-    def _read(self, field_names):
-        """ Read the given fields of the records in ``self`` from the database,
-            and store them in cache. Skip fields that are not stored.
-
-            :param field_names: list of field names to read
+    def fetch(self, field_names):
```

Comment:
```suggestion
    def _fetch(self, field_names):
```
`_fetch` should be private : it has no use in rpc that I can see